src/test/scala/nl/hugo/redbook/ch6/CNG.scala

```diff
@@ -0,0 +1,24 @@
+package nl.hugo.redbook.ch6
+
+object CNG {
```

Comment:
What doe CNG mean? Constant Number Generator? (RNG is already a stretch, I've only left it in because the book uses it.) I think that RNGMock or something similar covers the intent a lot better.